packages/@aws-cdk/region-info/build-tools/fact-tables.ts

```diff
@@ -26,6 +26,7 @@ export const AWS_CDK_METADATA = new Set([
   'eu-south-1',
   'me-south-1',
   'sa-east-1',
+  // 'ap-south-2',
```

Comment:
```suggestion
  'ap-south-2',
```

Reply:
No metadata resource in this region yet.
It's (apparently) a convention in this file to add the region commented out in this case.